static/js/common/DropdownMenu.jsx

```diff
@@ -44,10 +44,10 @@ const DropdownMenuItemWithIcon = ({icon, textEn, textHe}) => {
   );
 }
 
-const DropdownMenu = ({children, buttonComponent}) => {
+const DropdownMenu = ({children, buttonComponent, positioningClass}) => {
     /**
      * buttonComponent is a React component for the opening/closing of a button.
-     * the menu will be closed in click anywhere except in an element with classname 'preventClosing'.
+     * the menu will be closed in click anywhere except in an element where data attribute
```

Comment:
Please finish this sentance :-) (which data attribute, and that it needs to `="true"`